enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -496,6 +644,55 @@ the updates, and continue enforcement rollout only when the policy on previous c
   }
   ```
 
+#### Development Phase 4
+
+- **Summary**
+
+  - Implement rollouts for policy sets.
+
+- **Details**
+
+  - Add an `OrderPolicies` boolean and the `RolloutStrategy` struct to the spec of the `PolicySet`.
+  - `OrderPolicies` signals to the `governance-policy-propagator` controller whether to inject a dependency on the
+    previous policy in the set when propagating policies to the managed cluster namespaces.
+  - Logic would need to be added to the `governance-policy-propagator` controller to detect when a policy is bound to a
+    placement via a policy set to roll out policies in a set per cluster. The `GetClusterRolloutStatus` function should
+    be updated to consider all policies in the policy set for a particular cluster so that the library returns clusters
+    to roll out to based on the entire set of policies (rather than the individual policy). This flow looks like:
+    - Policies in the set across all managed cluster namespaces are set to `remediationAction: inform` upon update, as
+      specified in this enhancement, with the policy set overriding any `rolloutStrategy` specified in the individual
+      policy.
+    - When the policy set rollout is signaled by the `policy.open-cluster-management.io/last-rollout` annotation, the
+      propagator identifies whether a policy was bound to a placement via a policy set. From there:
```

Comment:
And the policy should only have the one placement defined.  That is the placement on the policyset in this case.  So if the policy is in another policyset too --- that 2nd policyset must use the same placement.  What if that policyset doesn't specify the rollout strategy?

Reply:
If the second policy set doesn't specify a rollout, I think the rollout would take precedence if it specifies the same cluster. But I'm not sure why you'd have the same policy in two separate sets with the same placement?